config/routes.rb

```diff
@@ -75,6 +75,7 @@
         get :text
       end
     end
+    resources :media_reserves, controller: "scanned_resources"
```

Comment:
I don't think this is needed — I think that the create route in https://github.com/pulibrary/figgy/blob/c511b407781fdb1b6969ff7c8ea20fb4a776f8bb/config/routes.rb#L90 is the only route that needs to be customized for media reserves, otherwise, the change_set can be looked up from the resource.